sdk/component/conf.go

```diff
@@ -228,42 +239,59 @@ func newRunnerConfig() (*RunnerConfig, error) {
 	// --- END - LOGGING ENV - END ---
 
 	// --- BEGIN - STORER ENV - BEGIN ---
-	st, err := fromEnvOrDefault(envVarKeyBackedStoreType, "", withFallbackToDefaultOnError(true))
+	st, err := fromEnvOrDefault(envVarKeyBackendStoreType, "", withFallbackToDefaultOnError(true))
 	if err != nil {
-		return nil, fmt.Errorf("could not lookup environment for '%s': %w", envVarKeyBackedStoreType, err)
+		return nil, fmt.Errorf("could not lookup environment for '%s': %w", envVarKeyBackendStoreType, err)
 	}
 
-	var (
-		storageType         = storeType(st)
-		store        Storer = nil
-		storeEnabled        = false
-	)
+	conf := &RunnerConfig{
+		ComponentName: componentName,
+		SDKVersion:    sdk.Version,
+		WorkflowID:    workflowID,
+		Logging: RunnerConfigLogging{
+			Level:  RunnerConfigLoggingLevel(logLevel),
+			Logger: logger,
+		},
+		PanicHandler: panicHandler,
+	}
 
 	if st != "" {
```

Comment:
we should use a better name for the storer implementation and we should also return immediately if it's an empty string. I think using an enum generator would help in this place because it would prevent us from ever having to deal with a random string value but we will always have to deal with enum values

Reply:
We're validating these contents here already https://github.com/smithy-security/smithy/blob/main/sdk/component/runner.go#L40!